winter/schema/inspectors.py

```diff
@@ -0,0 +1,29 @@
+import typing
+
+from drf_yasg.inspectors import SwaggerAutoSchema as SwaggerAutoSchemaBase
+
+from winter.routing import Route
+from winter.routing import get_function_route
+
+
+class SwaggerAutoSchema(SwaggerAutoSchemaBase):
+
+    def get_consumes(self):
+        route = self._get_route()
+        if route is None:
+            return []
```

Comment:
Is it ok? Maybe call base implementation in this case?

Reply:
Done